### PR TITLE
Bjorncs/travis

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

**Comment by bjorncs** [comment]
Fixes https://github.com/vespa-engine/vespa/issues/3435.

File: bootstrap-cmake.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    echo "Usage: $0 <source-dir> [<extra-cmake-args>]" >&2
+}
+
+if [[ $# -eq 1 ]]; then
+    SOURCE_DIR=$1
+    EXTRA_CMAKE_ARGS=""
+elif [ $# -eq 2 ]; then
+    SOURCE_DIR=$1
+    EXTRA_CMAKE_ARGS=$2
+elif [[ $# -eq 1 && ( "$1" = "-h" || "$1" = "--help" )]]; then
```
Comment (aressem) [inline bootstrap-cmake.sh]: You will never get here for help as the topmost if would be evaluated to true.

File: bootstrap-cmake.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+usage() {
+    echo "Usage: $0 <source-dir> [<extra-cmake-args>]" >&2
+}
+
+if [[ $# -eq 1 ]]; then
+    SOURCE_DIR=$1
+    EXTRA_CMAKE_ARGS=""
+elif [ $# -eq 2 ]; then
+    SOURCE_DIR=$1
+    EXTRA_CMAKE_ARGS=$2
```
Comment (aressem) [inline bootstrap-cmake.sh]: Would expect that I could pass multiple extra cmake args like -DMYDEF1=something -DMYDEF2=somethingelse.

**Review by aressem — CHANGES_REQUESTED** [review]
Reviewed

**Review by aressem — APPROVED** [review]

**Comment by aressem** [comment]
Merge when Travis is happy.

File: bootstrap-cmake.sh (hunk 2 above)
Comment (bjorncs) [reply bootstrap-cmake.sh]: Agree. Is there any easy way of varargs in bash?

Comment (aressem) [reply bootstrap-cmake.sh]:
```
SOURCE_DIR=$1
shift
EXTRA_CMAKE_ARGS="$@"
```

**Pull request merged**